client/my-sites/importer/importing-pane.jsx

```diff
@@ -116,7 +116,14 @@ class ImportingPane extends React.PureComponent {
 	getHeadingText = () => {
 		return translate(
 			'Importing may take a while if your site has a lot of media, but ' +
-				"you can safely navigate away from this page if you need to: we'll send you a notification when it's done."
+				'you can safely navigate away from this page if you need to or ' +
+				'go to our {{a}}handy checklist{{/a}} to explore your settings and customizing options. ' +
```

Comment:
`s/customizing/customization/`, imo

Reply:
Hey @jblz - do you mean you'd change customizing to customization? If so I 👍 that, I'll update!